src/uart/registers.rs

```diff
@@ -97,4 +97,84 @@ impl<const PORT_PTR: usize> ControlRegister<PORT_PTR> {
     /// - 0: Error detection is disabled
     /// - 1: Error detection is enabled
     is_bit_frame_error_detection_enabled}
+
+    bit_impl! {17..=16, RW u16,
```

Comment:
This range should also be flipped around, ranges for `bit_impl!` go from _low_ to _high_. 
```diff
- bit_impl! {17..=16, RW u16,
+ bit_impl! {16..=17, RW u16,
```

I would also recommend to return the smallest possible type that can fit these bits (ie. `u8`).